src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -154,414 +175,872 @@ module TaskSeq =
+    /// The given function will be applied as elements are demanded using the <see cref="MoveNextAsync" />
+    /// method on async enumerators retrieved from the input task sequence.
+    /// Does not evaluate the input sequence until requested.
+    /// </summary>
+    ///
+    /// <param name="binder">An asynchronous function to transform items from the input task sequence into a regular sequence.</param>
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting concatenation of all returned task sequences.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val collectSeqAsync: binder: ('T -> #Task<'SeqU>) -> source: taskSeq<'T> -> taskSeq<'U> when 'SeqU :> seq<'U>
 
     /// <summary>
-    /// Returns the first element of the task sequence from <paramref name="source" />, or <see cref="None" /> if the sequence is empty.
+    /// Returns the first element of the input task sequence given by <paramref name="source" />,
+    /// or <see cref="None" /> if the sequence is empty.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The first element of the task sequence, or <see cref="None" />.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val tryHead: source: taskSeq<'T> -> Task<'T option>
 
     /// <summary>
-    /// Returns the first elementof the task sequence from <paramref name="source" />
+    /// Returns the first elementof the input task sequence given by <paramref name="source" />.
     /// </summary>
-    /// <exception cref="ArgumentException">Thrown when the sequence is empty.</exception>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The first element of the task sequence.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
+    /// <exception cref="T:ArgumentException">Thrown when the task sequence is empty.</exception>
     val head: source: taskSeq<'T> -> Task<'T>
 
     /// <summary>
-    /// Returns the whole task sequence from <paramref name="source" />, minus its first element, or <see cref="None" /> if the sequence is empty.
+    /// Returns the whole input task sequence given by <paramref name="source" />, minus its first element,
+    /// or <see cref="None" /> if the sequence is empty.
     /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The input task sequence minus the first element, or <see cref="None" />.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     val tryTail: source: taskSeq<'T> -> Task<taskSeq<'T> option>
 
     /// <summary>
     /// Returns the whole task sequence from <paramref name="source" />, minus its first element.
```

Comment:
Again there is the Views/Builds/Returns/decorates thing - not sure that Return is particularly useful as its lazy

Maybe "Builds a task sequence that yields all but the first item of the source"

Reply:
This is done for parity with seq. We should improve the docs there before doing it here.